install_config/install/advanced_install.adoc

```diff
@@ -1745,6 +1745,22 @@ endif::[]
 These settings create a persistent volume that is attached to the OAB's etcd
 instance during cluster installation.
 
+[[configuring-oab-local-apb-devel]]
+==== Configuring the OpenShift Ansible Broker for Local APB Development
+
+In order to do xref:../../apb_devel/index.adoc#apb-devel-intro[APB development]
+with the OpenShift Container Registry in conjunction with the OAB, a whitelist
```

Comment:
Do we need to say here _a whitelist of images that OAB can access..._ (or similar)?